src/engraving/dom/gradualtempochange.cpp

```diff
@@ -359,6 +359,7 @@ TempoText* GradualTempoChangeSegment::findElementToSnapAfter() const
     // It is the next tempoChange which looks for a tempoChange before.
     Fraction refTick = tempoChange()->tick2();
     Measure* measure = score()->tick2measure(refTick);
+    measure = style().styleB(Sid::createMultiMeasureRests) && measure->hasMMRest() ? measure->mmRest() : measure;
```

Comment:
Sorry @miiizen I've slightly overlooked this one, I should have noticed before merging. This line is potentially unsafe cause you're using `measure` before null-checking it. So either you null-check it before using it, or you do directly 
Measure* measure = score()->tick2measureMM(refTick);
That function, if I'm not mistaken, returns measure->mmRest() if exist of measure otherwise, which is what you need here.

Reply:
Ah, I'll change it to `tick2measureMM`.